database.sql

```diff
@@ -797,55 +797,6 @@ CREATE TABLE IF NOT EXISTS `item` (
 	FOREIGN KEY (`psid`) REFERENCES `permissionset` (`id`) ON UPDATE RESTRICT ON DELETE RESTRICT
 ) DEFAULT COLLATE utf8mb4_general_ci COMMENT='Structure for all posts';
 
---
--- TABLE item-activity
---
-CREATE TABLE IF NOT EXISTS `item-activity` (
-	`id` int unsigned NOT NULL auto_increment,
-	`uri` varchar(255) COMMENT '',
-	`uri-id` int unsigned COMMENT 'Id of the item-uri table entry that contains the item uri',
-	`uri-hash` varchar(80) NOT NULL DEFAULT '' COMMENT 'RIPEMD-128 hash from uri',
-	`activity` smallint unsigned NOT NULL DEFAULT 0 COMMENT '',
-	 PRIMARY KEY(`id`),
-	 UNIQUE INDEX `uri-hash` (`uri-hash`),
-	 INDEX `uri` (`uri`(191)),
-	 INDEX `uri-id` (`uri-id`),
-	FOREIGN KEY (`uri-id`) REFERENCES `item-uri` (`id`) ON UPDATE RESTRICT ON DELETE CASCADE
-) DEFAULT COLLATE utf8mb4_general_ci COMMENT='Activities for items';
```

Comment:
Is this deletion intended?

Reply:
Yeah. This table was unused for some time now.